lib/sorteios_web/live/room_live/show.ex

```diff
@@ -28,20 +28,28 @@ defmodule SorteiosWeb.RoomLive.Show do
         |> EQRCode.encode()
         |> EQRCode.svg(width: 240)
 
-      {:ok,
```

Comment:
I dont think this change is needed?

Reply:
Somehow the `|> assign(:loading?, false)` wasn't working.